test/extensions/filters/network/redis_proxy/proxy_filter_test.cc

```diff
@@ -292,5 +297,78 @@ TEST_F(RedisProxyFilterTest, ProtocolError) {
   EXPECT_EQ(1UL, store_.counter("redis.foo.downstream_cx_protocol_error").value());
 }
 
-} // namespace Redis
+TEST(RedisProxyFilterConfigFactoryTest, RedisProxyCorrectJson) {
```

Comment:
Should this be its own test?  We split the config.cc into its own file.  I like 1:1 source files / test files but there is a completely arguable trade-off with extra linking / testing time to be made.

Reply:
I was being lazy here. Yes it should. You got me. I will clean this up (and other similar things in a follow up). 😉